### PR TITLE
Add ability to load only some countries based on config loadedCountries

### PR DESCRIPTION
**Pull request by aeq-dev**
Is your feature request related to a problem? Please describe.
No

Describe the solution you'd like
Sometimes we may need only some countries for our application, so no need to insert the whole World DB, and later we may append other countries

### CODE REVIEW

**Comment by nnjeim** [comment]
hi @aeq-dev thank you for your contribution. if you don't mind consider adding in config an additional key:
restrict_countries = false by default in order not to change the behaviour for the rest of the users.
in this case your condition in the seeder would be changed to include the restrict_countries too and the restriction array.
Please let me know what do you think of this?

File: src/Actions/SeedAction.php [hunk 1]
```diff
@@ -133,6 +133,10 @@ private function initCountries(): void
 	{
 		app(Models\Country::class)->truncate();
 		$this->countries['data'] = json_decode(File::get(__DIR__ . '/../../resources/json/countries.json'), true);
+		if (!empty(config('world.loadedCountries')))
```
Comment (nnjeim) [inline src/Actions/SeedAction.php:136]: if (config('world.restrictCountries') === true && !empty(config('world.loadedCountries')))

File: config/world.php [hunk 2]
```diff
@@ -1,6 +1,13 @@
 <?php
 
 return [
+	/*
+	|--------------------------------------------------------------------------
+	| Loaded Countries
+	|--------------------------------------------------------------------------
+	*/
+	'loadedCountries' => ['US', 'DZ'],
+
```
Comment (nnjeim) [inline config/world.php:10]: 'restrictCountries' => false,

**Comment by aeq-dev** [comment]
Hi @nnjeim 
Thanks for your consideration.
I think the '**loadedCountries**' key is enough, just mention in comment to leave the array empty if you'd like to migrate all countries, of course the default value of this key is empty [].

**Comment by nnjeim** [comment]
Hi again, in this case let's keep this array empty in order not to limit the usage for other users.
what do you think if we elaborate here by having:
allowed_countries and disallowed_countries arrays (like white and black listing)
these 2 arrays could be left empty by default.

**Review by nnjeim — APPROVED** [review]

**Comment by nnjeim** [comment]
> Yes good idea, Done.

Thanks a lot. Great addition.

**Pull request merged**